draw_flip_book/lib/flip_book_painter.dart

```diff
@@ -24,9 +24,9 @@ class FlipBookPainter extends CustomPainter {
     }
   }
 
-  bool shouldDrawPoint(int i) => offsets[i] != null && offsets[i + 1] == null;
+  bool shouldDrawPoint(int i) => offsets[i] != null && offsets.length > i + 1 && offsets[i + 1] == null;
```

Comment:
Added this to prevent `RangeError`s from showing up

Reply:
Hahaha I saw those